### ISSUE TITLE
Missing details endpoint for infrastructure services

### ISSUE THREAD

**Issue by lamppu** [eclipse-apoapsis/ort-server#758]
There is currently no GET `/api/v1/organizations/{organizationId}/infrastructure-services/{serviceName}` endpoint, which would be convenient for example when fetching existing values for an edit form in the frontend.

**Comment by sschuberth** [comment]
@MarcelBochtler, @oheger-bosch or @wkl3nk, is this something you could implement while @mnonnenmacher is still away?

**Comment by lamppu** [comment]
Yes, it's been resolved in https://github.com/eclipse-apoapsis/ort-server/pull/3359.
